requirements-client.txt

```diff
@@ -18,7 +18,7 @@ orjson >= 3.7, < 4.0
 packaging >= 21.3, < 24.3
 pathspec >= 0.8.0
 # https://github.com/PrefectHQ/prefect/issues/11619
-pendulum < 3.0; python_version < '3.12'
+pendulum < 4.0; python_version < '3.12'
```

Comment:
Interestingly enough pendulum requires >= 3.9 but builds 3.10 wheels 😅

Maybe you want to prevent Dependabot from upgrading to pendulum 3 if you think it breaks Python < 3.12.

By the way, pendulum 2 is 5 years old, pendulum 3 nearly 2 years.